Source/WebKit/GPUProcess/webrtc/LibWebRTCCodecsProxy.h

```diff
@@ -67,14 +68,14 @@ class SharedVideoFrameReader;
 class LibWebRTCCodecsProxy final : public IPC::WorkQueueMessageReceiver {
     WTF_MAKE_TZONE_ALLOCATED(LibWebRTCCodecsProxy);
 public:
-    static Ref<LibWebRTCCodecsProxy> create(GPUConnectionToWebProcess&);
+    static Ref<LibWebRTCCodecsProxy> create(GPUConnectionToWebProcess&, SharedPreferencesForWebProcess&);
     ~LibWebRTCCodecsProxy();
```

Comment:
Should be SharedPreferencesForWebProcess& or SharedPreferencesForWebProcess

Reply:
Nit: `LibWebRTCCodecsProxy` does not need `GPUConnectionToWebProcess`, so the constructor can just take `Connection`. It seems better to avoid passing `GPUConnectionToWebProcess&` while `GPUConnectionToWebProcess` is being constructed.

Reply:
Changed to be SharedPreferencesForWebProcess&.